ga4gh/datamodel/datasets.py

```diff
@@ -201,6 +246,7 @@ class FileSystemDataset(AbstractDataset):
     """
     variantsDirName = "variants"
     readsDirName = "reads"
+    featuresDirName = "sequenceAnnotations"
```

Comment:
That's a good question - it depends on whether we'll decide to host the wiggle data in the same directory, or whether to put them in a parallel structure. Now that I think about it, it may make sense to preserve the current `sequenceAnnotations` directory, and introduce two subdirectories, `features` and `continuous` for the different types of data that will eventually be served up.